scripts/qa_crowbarsetup.sh

```diff
@@ -4586,6 +4586,14 @@ function onadmin_prepare_crowbar_upgrade()
         # move nodes to upgrade mode
         safely curl -s -X POST $crowbar_api_digest $crowbar_api/installer/upgrade/prepare.json
     else
+        if crowbarctl upgrade prechecks --format plain | grep "false" ; then
+            complain 11 "Some necessary check before the upgrade has failed"
+        fi
```

Comment:
you could even find out that way which check failed (with another ruby command of course)

```bash
crowbarctl upgrade prechecks --format json | rubyjsonparse 'j.keys.each { |val| puts "#{val} failed" unless j[val] }'
```